packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/link_type/resolve.rs

```diff
@@ -44,13 +44,22 @@ where
                         let versioned_uri = VersionedUri::from_str(link_type_id)
                             .into_report()
                             .change_context(ResolveError::Custom)?;
-                        if let Some(entity_type_ref) = self.get(&versioned_uri) {
-                            context
-                                .read_versioned_ontology_type::<EntityType>(entity_type_ref.uri())
-                                .await
-                                .change_context(ResolveError::StoreReadError)?
-                                .resolve(tail_path_segments, context)
-                                .await
+                        if let Some(entity_type_ids) = self.get(&versioned_uri) {
```

Comment:
Any specific reason why you renamed it to `ids`? The return type here is `EntityTypeReference`. I don't mind the rename but then we should do that in other places as well.

Reply:
Good catch, thank you! It was a mistake while trying to figure out the new stream handling and flattening with you.